sdk/web-pubsub/web-pubsub/review/web-pubsub.api.md

```diff
@@ -0,0 +1,153 @@
+    // Warning: (ae-forgotten-export) The symbol "GroupSendTextToAllOptions" needs to be exported by the entry point index.d.ts
+    sendToAll(message: string, options: GroupSendTextToAllOptions): Promise<RestResponse>;
+    // Warning: (ae-forgotten-export) The symbol "JSONTypes" needs to be exported by the entry point index.d.ts
+    // Warning: (ae-forgotten-export) The symbol "GroupSendToAllOptions" needs to be exported by the entry point index.d.ts
+    sendToAll(message: JSONTypes, options?: GroupSendToAllOptions): Promise<RestResponse>;
+    sendToAll(message: HttpRequestBody, options?: GroupSendToAllOptions): Promise<RestResponse>;
+}
+
+// @public
+export class WebPubSubServiceClient {
+    constructor(connectionString: string, hubName: string, options?: HubAdminClientOptions);
+    constructor(endpoint: string, credential: AzureKeyCredential, hubName: string, options?: HubAdminClientOptions);
+    readonly apiVersion: string;
+    closeConnection(connectionId: string, options?: CloseConnectionOptions): Promise<RestResponse>;
+    endpoint: string;
+    getAuthenticationToken(options?: GetAuthenticationTokenOptions): Promise<GetAuthenticationTokenResponse>;
+    // Warning: (ae-forgotten-export) The symbol "Permission" needs to be exported by the entry point index.d.ts
+    // Warning: (ae-forgotten-export) The symbol "HubGrantPermissionOptions" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    grantPermission(connectionId: string, permission: Permission, options?: HubGrantPermissionOptions): Promise<RestResponse>;
+    group(groupName: string): WebPubSubGroup;
+    hasConnection(connectionId: string, options?: HasConnectionOptions): Promise<boolean>;
+    hasGroup(groupName: string, options?: HubHasGroupOptions): Promise<boolean>;
+    hasUser(username: string, options?: HubHasUserOptions): Promise<boolean>;
+    readonly hubName: string;
+    removeUserFromAllGroups(userId: string, options?: CloseConnectionOptions): Promise<RestResponse>;
+    // Warning: (ae-forgotten-export) The symbol "HubRevokePermissionOptions" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    revokePermission(connectionId: string, permission: Permission, options?: HubRevokePermissionOptions): Promise<RestResponse>;
+    // Warning: (ae-forgotten-export) The symbol "HubSendTextToAllOptions" needs to be exported by the entry point index.d.ts
+    sendToAll(message: string, options: HubSendTextToAllOptions): Promise<RestResponse>;
+    sendToAll(message: JSONTypes, options?: HubSendToAllOptions): Promise<RestResponse>;
+    sendToAll(message: HttpRequestBody, options?: HubSendToAllOptions): Promise<RestResponse>;
+    // Warning: (ae-forgotten-export) The symbol "HubSendTextToConnectionOptions" needs to be exported by the entry point index.d.ts
+    sendToConnection(connectionId: string, message: string, options: HubSendTextToConnectionOptions): Promise<RestResponse>;
```

Comment:
are these `connectionId`s the same as the ones from groups? Can two connections have the same user? At least from the naming it's very fuzzy the relationship between connections and users

Reply:
Yes, same one. A user can have multiple connections (for example, if the same user is connected on their PC and their phone).